bin/ch-test

```diff
@@ -935,6 +935,13 @@ if [[ -z $phase ]]; then
 fi
 printf '\n'
 
+# FIXME: https://github.com/hpc/charliecloud/issues/1580
+# shellcheck disable=SC2016
+echo  'NOTE: default image and pack directories changed to:'
+echo  '      CH_TEST_IMGDIR=/var/tmp/${USER}.img'
+echo  '      CH_TEST_TARDIR=/var/tmp/${USER}.pack'
+printf '\n'
```

Comment:
Might be good to print this (1) to stderr and (2) only if the old defaults actually exist.